master/internal/experiment.go

```diff
@@ -42,11 +42,22 @@ type (
 	trialQueryIsBestValidation struct {
 		validationMetrics workload.ValidationMetrics
 	}
+
+	// Searcher-related messages.
+	trialTrainUntilReq struct {
+		trialID int
+	}
+	trialTrainUntilResp struct {
+		finished bool
+		length   model.Length
+	}
+
 	// trialClosed is used to replay closes missed when the master dies between when a trial closing in
 	// its actor.PostStop and when the experiment snapshots the trial closed.
 	trialClosed struct {
 		requestID model.RequestID
 	}
+	// TODO(brad): This message is redundant.
```

Comment:
Ah yeah, it's _not_ redundant, I've just been doing this "find the trial actor" in two different ways.. this is probably the right way, instead of just sending to a non-existent actor based on some persistent metadata to find its addr